flowmachine/flowmachine/core/server/query_schemas/flows.py

```diff
@@ -49,13 +50,15 @@ def _flowmachine_query_obj(self):
         """
         loc1 = self.from_location._flowmachine_query_obj
         loc2 = self.to_location._flowmachine_query_obj
-        return RedactedFlows(flows=Flows(loc1, loc2))
+        return RedactedFlows(flows=Flows(loc1, loc2, join_type=self.join_type))
 
 
 class FlowsSchema(BaseSchema):
+    valid_joins = ("inner", "full outer", "left", "right", "left outer", "right outer")
```

Comment:
This is == those valid for `Join`, right? Might make sense to put those there, and validate the input to join type (still want to validate it here of course).